src/Reveal.Sdk.Dom.Tests/Visualizations/VisualizationFixture.cs

```diff
@@ -0,0 +1,445 @@
+﻿using Xunit;
+using Moq;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Visualizations;
+using System.Collections.Generic;
+using Xunit.Sdk;
+using Newtonsoft.Json.Linq;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Reveal.Sdk.Dom.Core.Utilities;
+using System;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+    public class VisualizationFixture
+    {
+        [Fact]
+        public void Constructor_SetTitleAndChartType_WhenConstructed()
+        {
+            // Arrange
+            var dataSource = new DataSource();
+            var dataSourceItem = new DataSourceItem { DataSource = dataSource };
+            var testTitle = "testTitle";
+
+            // Act
+            var visualization = new Mock<Visualization>(testTitle, dataSourceItem) { CallBase = true }.Object;
+
+            // Assert
+            Assert.Equal(testTitle, visualization.Title);
```

Comment:
There are calls to `InitializeDataDefinition` and `UpdateDataSourceItem`, so I think we need more asserts with default values.
For e.x: Default value of `HasTabularData` field in DataSourceItem is true. So we need `Assert.IsType<TabularDataDefinition>(visualization.DataDefinition);` assertion

Reply:
This test actually doesn't require data source Item, so I will set it to null, only need to check title and chart type.

For default cases when has data source, it depends on the dsItem's type, and there are 2 cases for it